lib/net/pool.js

```diff
@@ -4214,6 +4264,10 @@ class PoolOptions {
     this.port = this.network.port;
     this.publicHost = '0.0.0.0';
     this.publicPort = this.network.port;
+    this.brontideHost = '0.0.0.0';
+    this.brontidePort = this.network.brontidePort;
+    this.publicBrontideHost = '0.0.0.0';
+    this.publicBrontidePort = this.network.brontidePort;
```

Comment:
These options may need to be added to `FullNode` and `SPVNode` as well so they can be set from the config file, etc.

Reply:
These options were added to `FullNode` but there is no need to add them to `SPVNode`, as it is configured with `listen: false` so it will not accept incoming connections.

https://github.com/handshake-org/hsd/blob/master/lib/node/spvnode.js#L76
https://github.com/handshake-org/hsd/blob/6afb18b134f97ec702b268a52f7673b8aa4e132b/lib/net/pool.js#L366-L367

Reply:
See 0e3da111539d67283c529b781310524db2480920

Reply:
re: SPV -- oh yeah! 👍